crates/sui-bridge/src/node.rs

```diff
@@ -165,7 +165,8 @@ async fn start_watchdog(
         _config_address,
         weth_address,
         usdt_address,
-        _wbtc_address,
+        wbtc_address,
+        lbtc_address,
     ) = get_eth_contract_addresses(eth_bridge_proxy_address, &eth_provider)
```

Comment:
`get_eth_contract_addresses` needs to be updated to return the lbtc address I think